Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -111,28 +111,24 @@ void App::Connect(int connectionId, Platform::Object^ target) {
 }
 
 void App::OnLaunched(LaunchActivatedEventArgs^ args) {
-    if (EbrApplicationLaunched(args) == true) {
-        _ApplicationMainLaunch(ActivationTypeNone, nullptr);
-    }
+    EbrApplicationActivated(args);
```

Comment:
Have you tested suspend/resume with these changes?

---
In reply to: [72308770](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72308770) [](ancestors = 72308770)